database/migrations/2022_06_18_081620_add_columns_task_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('tasks', function (Blueprint $table) {
+            $table->string('date', 8);
+            $table->string('time', 2);
+            $table->unique(['date', 'time']);
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('tasks', function (Blueprint $table) {
+            $table->dropColumn('date');
+            $table->dropColumn('time');
+            $table->dropUnique('tasks_date_time_unique');
```

Comment:
I don't think this works have you tested it?

Reply:
yes i have tested it 
first i run the command `php artisan migrate:rollback`
then i run `php artisan migrate`
and my file migrated